src/main/java/it/smartcommunitylab/aac/webauthn/service/WebAuthnRpService.java

```diff
@@ -0,0 +1,241 @@
+package it.smartcommunitylab.aac.webauthn.service;
+
+import java.security.SecureRandom;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+import com.ibm.icu.impl.Pair;
```

Comment:
remove usage of this class

Reply:
Removed in f5dda8a98fb396dccffe3f21455aa6b931bbd3f6